packages/@ember/application/lib/application.js

```diff
@@ -36,46 +36,32 @@ let librariesRegistered = false;
 
 /**
   An instance of `Application` is the starting point for every Ember
-  application. It helps to instantiate, initialize and coordinate the many
+  application. instantiates, initializes and coordinates the
   objects that make up your app.
 
-  Each Ember app has one and only one `Application` object. In fact, the
-  very first thing you should do in your application is create the instance:
+  Each Ember app has one and only one `Application` object. Although
+  Ember CLI creates this object implicitly, the `Application` class
+  is defined in the `app/app.js`. You can define a `ready` method on the
+  `Application` class, which will be run by Ember when the application is
+  initialized.
 
   ```javascript
-  import Application from '@ember/application';
-
-  window.App = Application.create();
-  ```
-
-  Typically, the application object is the only global variable. All other
-  classes in your app should be properties on the `Application` instance,
-  which highlights its first role: a global namespace.
-
-  For example, if you define a view class, it might look like this:
-
-  ```javascript
-  import Application from '@ember/application';
-
-  App.MyView = Ember.View.extend();
+  let App = Application.extend({
```

Comment:
```suggestion
  const App = Application.extend({
```

Reply:
awesome :) I was going to convert all these to consts, but I think i've seen some people prefer `let` as a rule, so didn't want to bring that up. I'm +1 for this.